libV2/server/middleware.js

```diff
@@ -119,7 +119,7 @@ async function authV4Middleware(request, response, params) {
     } catch (error) {
         request.logger.error('error during authentication', { error });
         // rethrow any access denied errors
-        if (error.AccessDenied) {
+        if (error && (error.is || {}).AccessDenied) {
```

Comment:
no big deal, but it could be better to be consistent in how we check errors regarding the new style, i.e. choosing a coding style, like compared to https://github.com/scality/utapi/pull/1243/files#diff-664226905101de232ca7df5e0172bc90bbf7cf1e3b26a64e320b37ee0713dfbdR106

Also here I guess the `error &&` is not needed since we are in the catch block